src/components/Logout/index.tsx

```diff
@@ -12,14 +13,20 @@ export interface LogoutProps {
  * url of the authentication server in order to go around the browser's CORS policy.
  */
 export const logoutFromAuthServer = (logoutURL: string) => {
-  const logoutWindow: Window | null = window.open(logoutURL);
-  const timer: NodeJS.Timeout = setInterval(() => {
-    if (logoutWindow) {
-      logoutWindow.close();
-      window.location.href = EXPRESS_OAUTH_LOGOUT_URL;
-    }
-    clearInterval(timer);
-  }, 20);
+  const logoutWindow = window.open(logoutURL);
+  /** I am not sure the optimum value for this, mozilla firefox seems to take some
+   * time loading up the logoutWindow and actually making the request.
+   */
+  const RedirectAfter = 1000;
```

Comment:
Should we make it configurable?  (it should at least be a constant)

Reply:
yeah